realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -77,6 +88,7 @@
     protected SharedGroupManager sharedGroupManager;
     protected boolean autoRefresh;
     Handler handler;
+    ColumnIndices columnIndices;
```

Comment:
`DynamicRealm` does not assign `ColumnIndices` instance to `columnIndices` and thus leave it `null`.